jetty-proxy/src/test/java/org/eclipse/jetty/proxy/AsyncMiddleManServletTest.java

```diff
@@ -1122,14 +1170,15 @@ public long getLength()
                     return data.length + 1;
                 }
             })
-            .timeout(5 * idleTimeout, TimeUnit.MILLISECONDS)
+            .timeout(15, TimeUnit.SECONDS)
```

Comment:
Was `5 * idleTimeout` so it should be `15 * idleTimeout` now.

Reply:
Based on `15 * idleTimeout` now